relatio/_embeddings.py

```diff
@@ -98,29 +101,39 @@ def use_sif(self) -> bool:
         return self._use_sif
 
     # One cannot add a setter since it is added next to the child classes
-
     def get_vector(self, phrase: str) -> Optional[np.ndarray]:
         tokens = phrase.split()
 
-        # For phrases with one token
-        if self.use_sif is True and len(tokens) > 0:
-
+        if self.use_sif:
```

Comment:
Should we do this for one token? 
What is more expensive: to get the `len` or to do the entire code block for this case?

Reply:
I would argue it's mainly for the consistency of the "new" embeddings space we're creating (not for speed)